### DISCUSSION TITLE
Visualizing NER entities highlights withing the tool

### DISCUSSION THREAD

**Discussion by fillipefbr** [sepinf-inc/IPED#449] [Q&A] [answered]
Hello!

I couldn't find a way to enable Named Entity Recognition functionality to work within the IPED-SearchApp.exe. 
I have already downloaded Standford's CoreNLP for English and put it in the optional_jars folder. I also edited the IPEDConfig.txt (from profiles/en/default and profiles/pt-BR/default) to enable NER and I get to see its task running successfully when generating the output. However, I can't find an option to visually run the NER module over a PDF or text document, for example.
Am I missing something?

Thanks.

**Comment by lfcnassif** [comment, marked as answer]
Take a look at: the new columns created (possibly hidden, set visibility on column manager), the third vertical subtab of the metadata viewer, and at the NER property group of the Metadata filtering panel:

![image](https://user-images.githubusercontent.com/7276994/114118117-940cb100-98be-11eb-8659-f21c7879fde1.png)

![image](https://user-images.githubusercontent.com/7276994/114118163-b1da1600-98be-11eb-878c-ecfc29714687.png)

**Reply by fillipefbr** [reply]
Got it now, thank you!
